collection-scripts/gather_logs

```diff
@@ -34,13 +34,43 @@ for ns in ${namespaces[@]}; do
     done
   done
 
+  for migplan in $(oc get migplan --namespace ${ns} --no-headers | awk '{print $1}'); do
+    echo ${migplan}
+    mkdir -p "/must-gather/migplan-${migplan}"
+    oc get migplan ${migplan} -o yaml --namespace ${ns} &> "/must-gather/migplan-${migplan}/${migplan}.yml"
+  done
+
+  for migmigration in $(oc get migmigration --namespace ${ns} --no-headers | awk '{print $1}'); do
```

Comment:
Our goal here is to nest the migmigrations associated with a migplan under that migplans directory. To do this, will need to parse the content of the migmigration and check what the associated migplan is.